osu.Game/Screens/Edit/Compose/Components/SelectionBoxDragHandleDisplay.cs

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using System.Linq;
+using JetBrains.Annotations;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+
+namespace osu.Game.Screens.Edit.Compose.Components
+{
+    /// <summary>
+    /// Represents a display composite containing and managing the visibility state of the selection box's drag handles.
+    /// </summary>
+    public class SelectionBoxDragHandleDisplay : CompositeDrawable
```

Comment:
not sold on this name. maybe `SelectionBoxDragHandleContainer` to clarify what's the difference between this and a `SelectionBoxDragHandle`

Reply:
Wasn't sure about using the suffix `Container` on a composite at first, but I guess it's fine.